waspc/data/Generator/templates/react-app/src/crud/_crud.ts

```diff
@@ -0,0 +1,99 @@
+{{={= =}=}}
+import { createAction } from "../actions/core";
+import { useAction } from "../actions";
+import { createQuery } from "../queries/core";
+import { useQuery } from "../queries";
+import { 
+    {=# operations.Get =}
+    GetQueryResolved,
+    {=/ operations.Get =}
+    {=# operations.GetAll =}
+    GetAllQueryResolved,
+    {=/ operations.GetAll =}
+    {=# operations.Create =}
+    CreateActionResolved,
+    {=/ operations.Create =}
+    {=# operations.Update =}
+    UpdateActionResolved,
+    {=/ operations.Update =}
+    {=# operations.Delete =}
+    DeleteActionResolved,
+    {=/ operations.Delete =}
+} from '../../../server/src/crud/{= name =}'
+
+function createCrud() {
+    {=# operations.Get =}
+    const crudGetQuery = createQuery<GetQueryResolved>(
+        '{= fullPath =}',
+        {=& entitiesArray =}
+    )
+    {=/ operations.Get =}
+    {=# operations.GetAll =}
+    const crudGetAllQuery = createQuery<GetAllQueryResolved>(
+        '{= fullPath =}',
+        {=& entitiesArray =}
+    )
+    {=/ operations.GetAll =}
+    {=# operations.Create =}
+    const crudCreateAction = createAction<CreateActionResolved>(
+        '{= fullPath =}',
+        {=& entitiesArray =}
+    )
+    {=/ operations.Create =}
+    {=# operations.Update =}
+    const crudUpdateAction = createAction<UpdateActionResolved>(
+        '{= fullPath =}',
+        {=& entitiesArray =}
+    )
+    {=/ operations.Update =}
+    {=# operations.Delete =}
+    const crudDeleteAction = createAction<DeleteActionResolved>(
+        '{= fullPath =}',
+        {=& entitiesArray =}
+    )
+    {=/ operations.Delete =}
+    return {
+        {=# operations.Get =}
+        get: {
+            query: crudGetQuery,
+            useQuery(args: Parameters<GetQueryResolved>[0]) {
```

Comment:
This is a change from our current way of using queries and actions. Instead of going for:
```ts
import { tasks } from '@wasp/crud/tasks';
import { useQuery } from '@wasp/queries';
...
const { data } = useQuery(tasks.getAll)
```
users can do:
```ts
import { tasks } from '@wasp/crud/tasks';
...
const { data } = tasks.getAll.useQuery()
```

Reply:
Yeah, this is a bit different!

One could say that we should keep it consistent. And we could, right? So what shall we do? Should we switch both to this new style? Or should we make this be like it was before?
Or we go indeed with two separate styles (which is a tiny bit weird?)? Would love to discuss this a bit.

Reply:
I would like to introduce this new syntax to operations later on. It's easier to use IMHO

Reply:
I guess it makes sense! One part of me wants to say that we should keep it as it is now and do that change later, in one go, but I don't want to stiffle inovation :D. So let's do it now, and we can live for a bit with inconsistent API, but can you in that case pls create an issue for this on GH? Then we can get to this relatively soon and ensure it is all uniform.

Reply:
We already have an issue for this: https://github.com/wasp-lang/wasp/issues/1069

Reply:
Ok, just make sure to comment there that we are doing it with CRUD now so we should look into doing it for everything or not for anything.